src/main/java/configurations/ConfigReader.java

```diff
@@ -3,5 +3,7 @@
 import org.aeonbits.owner.ConfigFactory;
 
 public class ConfigReader {
-    public static final EmulatorConfig emulatorConfig = ConfigFactory.create(EmulatorConfig.class, System.getProperties());
+    public static final AndroidConfig ANDROID_CONFIG = ConfigFactory.create(AndroidConfig.class, System.getProperties());
```

Comment:
Why do you need separate configs for iOS and Android? Do the fields which are not common there chnage at all? what's the purpose of this config files?